src/fides/api/ops/models/privacy_experience.py

```diff
@@ -165,26 +297,302 @@ def update(self, db: Session, *, data: dict[str, Any]) -> PrivacyExperience:
         self.privacy_notices = self.get_related_privacy_notices(db)
         return self
 
-    @hybridproperty
-    def privacy_experience_history_id(self) -> Optional[str]:
-        """Convenience property that returns the historical privacy experience id for the current version.
+    @staticmethod
+    def get_experience_by_component_and_region(
+        db: Session, region: PrivacyNoticeRegion, component: ComponentType
+    ) -> Optional[PrivacyExperience]:
+        """Load an experience for a given region and component type"""
+        return (
+            db.query(PrivacyExperience)
+            .filter(
+                PrivacyExperience.region == region,
+                PrivacyExperience.component == component,
+            )
+            .first()
+        )
 
-        Note that there are possibly many historical records for the given experience, this just returns the current
-        corresponding historical record.
-        """
-        history: PrivacyExperienceHistory = self.histories.filter_by(  # type: ignore # pylint: disable=no-member
-            version=self.version
-        ).first()
-        return history.id if history else None
+    @staticmethod
+    def get_experiences_by_region(
+        db: Session, region: PrivacyNoticeRegion
+    ) -> Tuple[Optional[PrivacyExperience], Optional[PrivacyExperience]]:
+        """Load both the overlay and privacy center experience for a given region"""
+        overlay_experience: Optional[PrivacyExperience] = (
+            db.query(PrivacyExperience)
+            .filter(
+                PrivacyExperience.region == region,
+                PrivacyExperience.component == ComponentType.overlay,
+            )
+            .first()
```

Comment:
this .first() is just a formality, right? As in the db should validate uniqueness on region and overlay

Reply:
yes and further, I do need to return the first PrivacyExperience/None, not the "Query" object. What is nice about `first()` is that it returns the first result of the query, or None if it doesn't exist!